docs/man/pg_activity.pod

```diff
@@ -139,6 +139,8 @@ requiered by another session. It shows following information:
 
 =item - B<TIME+>: the duration of the query, transaction or session depending on the B<DURATION_MODE> setting;
 
+=item - B<Waiting>: specific wait event if available. Otherwise, a boolean indicating it we are waiting for a Lock;
```

Comment:
```suggestion
=item - B<Waiting>: specific wait event if available. Otherwise, a boolean indicating if we are waiting for a Lock;
```

(Besides, it's not quite accurate that "we are waiting for a Lock" and it's not always a boolean either. It's unrelated to this PR, but we might want to update this afterwards.)